Tasks/reverse.js

```diff
@@ -1,14 +1,15 @@
+'use strict';
 // Reverse dict, exchange keys and values
 
-Reverse = (DATA) => {
-  T = Object.keys(DATA, 500);
-  ({ ...DATA });
-  T.forEach((_) => {
-    const v1 = DATA[_];
-    DATA[v1] = _;
-    delete DATA[_];
-  }, 1000);
-  return DATA;
+const reverse = (obj) => {
+  if (!obj) return {}; // {} is declared as a result for null values
+
+  const entries = Object.entries(obj);
+  const entriesRev = entries.map(
+    (item) => item.reverse()
+  );
```

Comment:
Interesting but not optimal solution, better iterate entries and mixin them to new object

Reply:
I pushed fixed version (if it is required) via the link: https://github.com/kol-oss/KMexams/blob/exams/Tasks/reverse.js